2025-02-23 23:13:40.123 [info] Extension host with pid 3674 started
2025-02-23 23:13:40.123 [info] Skipping acquiring lock for /home/kh/.config/Code - OSS/User/workspaceStorage/81774d8ce8dc5d35c55c012fcbd96777.
2025-02-23 23:13:40.199 [info] ExtensionService#_doActivateExtension vscode.emmet, startup: false, activationEvent: 'onLanguage'
2025-02-23 23:13:40.215 [info] ExtensionService#_doActivateExtension KevinRose.vsc-python-indent, startup: false, activationEvent: 'onLanguage:python'
2025-02-23 23:13:40.217 [info] ExtensionService#_doActivateExtension ms-python.python, startup: false, activationEvent: 'onLanguage:python', root cause: ms-python.debugpy
2025-02-23 23:13:40.436 [info] ExtensionService#_doActivateExtension ms-python.debugpy, startup: false, activationEvent: 'onLanguage:python'
2025-02-23 23:13:40.477 [error] Activating extension ms-python.debugpy failed due to an error:
2025-02-23 23:13:40.477 [error] Error: Extension 'ms-python.debugpy' CANNOT use API proposal: portsAttributes.
Its package.json#enabledApiProposals-property declares:  but NOT portsAttributes.
 The missing proposal MUST be added and you must start in extension development mode or use the following command line switch: --enable-proposed-api ms-python.debugpy
	at N (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:110:16858)
	at Object.registerPortAttributesProvider (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:172:55654)
	at t.registerDebugger (/home/kh/.vscode-oss/extensions/ms-python.debugpy-2025.0.1-linux-x64/dist/extension.js:2:846258)
	at e.activate (/home/kh/.vscode-oss/extensions/ms-python.debugpy-2025.0.1-linux-x64/dist/extension.js:2:1097551)
	at Ow.kb (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:116:15214)
	at Ow.jb (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:116:14921)
	at file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:116:13079
	at async aw.n (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:112:4447)
	at async aw.m (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:112:4410)
	at async aw.l (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:112:3866)
2025-02-23 23:13:40.486 [info] ExtensionService#_doActivateExtension vscode.git-base, startup: true, activationEvent: '*', root cause: vscode.git
2025-02-23 23:13:40.498 [info] ExtensionService#_doActivateExtension vscode.git, startup: true, activationEvent: '*'
2025-02-23 23:13:40.551 [info] ExtensionService#_doActivateExtension vscode.github, startup: true, activationEvent: '*'
2025-02-23 23:13:40.596 [info] Eager extensions activated
2025-02-23 23:13:40.614 [info] ExtensionService#_doActivateExtension vscode.debug-auto-launch, startup: false, activationEvent: 'onStartupFinished'
2025-02-23 23:13:40.616 [info] ExtensionService#_doActivateExtension vscode.merge-conflict, startup: false, activationEvent: 'onStartupFinished'
2025-02-23 23:13:41.497 [error] Error: Extension 'ms-python.python' CANNOT use API proposal: testObserver.
Its package.json#enabledApiProposals-property declares:  but NOT testObserver.
 The missing proposal MUST be added and you must start in extension development mode or use the following command line switch: --enable-proposed-api ms-python.python
	at N (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:110:16858)
	at get testResults (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:172:39426)
	at C.activate (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:908459)
	at /home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:2457
	at Array.map (<anonymous>)
	at v.activateWorkspace (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:2447)
2025-02-23 23:13:43.286 [warning] [Decorations] CAPPING events from decorations provider vscode.git 10000
2025-02-23 23:13:43.330 [info] ExtensionService#_doActivateExtension vscode.github-authentication, startup: false, activationEvent: 'onAuthenticationRequest:github'
2025-02-23 23:13:44.517 [info] ExtensionService#_doActivateExtension vscode.npm, startup: false, activationEvent: 'onTerminalQuickFixRequest:ms-vscode.npm-command'
2025-02-23 23:13:44.981 [warning] [Decorations] CAPPING events from decorations provider vscode.git 10001
2025-02-23 23:14:31.529 [error] [ms-python.python] provider FAILED
2025-02-23 23:14:31.529 [error] Error: FileNotFoundError: [Errno 2] No such file or directory: '/home/kh/hyprcrow/Untitled-1'
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-23 23:14:32.524 [error] [ms-python.python] provider FAILED
2025-02-23 23:14:32.524 [error] Error: FileNotFoundError: [Errno 2] No such file or directory: '/home/kh/hyprcrow/Untitled-1'
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-23 23:14:37.298 [error] [ms-python.python] provider FAILED
2025-02-23 23:14:37.298 [error] Error: FileNotFoundError: [Errno 2] No such file or directory: '/home/kh/hyprcrow/Untitled-1'
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-23 23:14:37.930 [error] [ms-python.python] provider FAILED
2025-02-23 23:14:37.930 [error] Error: FileNotFoundError: [Errno 2] No such file or directory: '/home/kh/hyprcrow/Untitled-1'
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-23 23:14:38.424 [error] [ms-python.python] provider FAILED
2025-02-23 23:14:38.424 [error] Error: FileNotFoundError: [Errno 2] No such file or directory: '/home/kh/hyprcrow/Untitled-1'
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-23 23:14:38.986 [error] [ms-python.python] provider FAILED
2025-02-23 23:14:38.986 [error] Error: FileNotFoundError: [Errno 2] No such file or directory: '/home/kh/hyprcrow/Untitled-1'
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-23 23:14:39.286 [error] [ms-python.python] provider FAILED
2025-02-23 23:14:39.286 [error] Error: FileNotFoundError: [Errno 2] No such file or directory: '/home/kh/hyprcrow/Untitled-1'
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-23 23:14:39.695 [error] [ms-python.python] provider FAILED
2025-02-23 23:14:39.695 [error] Error: FileNotFoundError: [Errno 2] No such file or directory: '/home/kh/hyprcrow/Untitled-1'
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-23 23:14:40.565 [error] [ms-python.python] provider FAILED
2025-02-23 23:14:40.565 [error] Error: FileNotFoundError: [Errno 2] No such file or directory: '/home/kh/hyprcrow/Untitled-1'
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-23 23:15:41.073 [error] [ms-python.python] provider FAILED
2025-02-23 23:15:41.073 [error] Error: UnboundLocalError: cannot access local variable 'end_index' where it is not associated with a value
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-23 23:15:42.890 [error] [ms-python.python] provider FAILED
2025-02-23 23:15:42.890 [error] Error: UnboundLocalError: cannot access local variable 'end_index' where it is not associated with a value
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-23 23:15:44.430 [error] [ms-python.python] provider FAILED
2025-02-23 23:15:44.430 [error] Error: UnboundLocalError: cannot access local variable 'end_index' where it is not associated with a value
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-23 23:15:45.475 [error] [ms-python.python] provider FAILED
2025-02-23 23:15:45.475 [error] Error: UnboundLocalError: cannot access local variable 'end_index' where it is not associated with a value
	at ae (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1973990)
	at oe (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1971950)
	at Immediate.<anonymous> (/home/kh/.vscode-oss/extensions/ms-python.python-2025.0.0-universal/out/client/extension.js:2:1968244)
	at process.processImmediate (node:internal/timers:483:21)
2025-02-23 23:19:17.716 [info] Extension host terminating: renderer closed the MessagePort
2025-02-23 23:19:17.736 [error] Canceled: Canceled
	at new XR (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:29:110406)
	at Uy.U (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:29:115016)
	at s.<computed>.n.charCodeAt.s.<computed> (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:29:112510)
	at mw.g (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:112:32221)
	at mw.executeCommand (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:112:31725)
	at r.registerCommand.description (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:112:31055)
	at mw.h (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:112:32741)
	at mw.g (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:112:31818)
	at mw.executeCommand (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:112:31725)
	at Object.executeCommand (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:172:37254)
	at Ah.L [as value] (/usr/lib/code/extensions/git/dist/main.js:2:997602)
	at D.B (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:27:2373)
	at D.C (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:27:2443)
	at D.fire (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:27:2660)
	at Object.p [as dispose] (/usr/lib/code/extensions/git/dist/main.js:2:1020242)
	at /usr/lib/code/extensions/git/dist/main.js:2:1026621
	at Array.forEach (<anonymous>)
	at b.dispose (/usr/lib/code/extensions/git/dist/main.js:2:1026607)
	at cv.<anonymous> (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:106:20084)
	at cv.dispose (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:106:20180)
	at cv.<anonymous> (/usr/lib/code/extensions/git/dist/main.js:2:1000086)
	at cv.dispose (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:106:20180)
	at ui (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:25:699)
	at file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:116:14992
	at file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:25:1009
	at Object.dispose (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:8:705)
	at PW.eb (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:116:11919)
	at file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:116:9874
	at Array.map (<anonymous>)
	at PW.$ (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:116:9862)
	at PW.terminate (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:116:10105)
	at wC.terminate (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:117:1434)
	at ts (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:190:15080)
	at MessagePortMain.<anonymous> (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:190:11628)
	at MessagePortMain.emit (node:events:518:28)
	at MessagePortMain._internalPort.emit (node:electron/js2c/utility_init:2:2949) setContext undefined
2025-02-23 23:19:17.737 [error] Canceled: Canceled
	at new XR (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:29:110406)
	at Uy.U (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:29:115016)
	at s.<computed>.n.charCodeAt.s.<computed> (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:29:112510)
	at mw.g (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:112:32221)
	at mw.executeCommand (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:112:31725)
	at r.registerCommand.description (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:112:31055)
	at mw.h (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:112:32741)
	at mw.g (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:112:31818)
	at mw.executeCommand (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:112:31725)
	at Object.executeCommand (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:172:37254)
	at set hasGitHubRepositories (/usr/lib/code/extensions/github/dist/extension.js:2:344164)
	at /usr/lib/code/extensions/github/dist/extension.js:2:345123
	at Ah.value (/usr/lib/code/extensions/git/dist/main.js:2:1117802)
	at D.B (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:27:2373)
	at D.C (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:27:2443)
	at D.fire (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:27:2660)
	at Object.p [as dispose] (/usr/lib/code/extensions/git/dist/main.js:2:1020242)
	at /usr/lib/code/extensions/git/dist/main.js:2:1026621
	at Array.forEach (<anonymous>)
	at b.dispose (/usr/lib/code/extensions/git/dist/main.js:2:1026607)
	at cv.<anonymous> (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:106:20084)
	at cv.dispose (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:106:20180)
	at cv.<anonymous> (/usr/lib/code/extensions/git/dist/main.js:2:1000086)
	at cv.dispose (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:106:20180)
	at ui (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:25:699)
	at file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:116:14992
	at file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:25:1009
	at Object.dispose (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:8:705)
	at PW.eb (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:116:11919)
	at file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:116:9874
	at Array.map (<anonymous>)
	at PW.$ (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:116:9862)
	at PW.terminate (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:116:10105)
	at wC.terminate (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:117:1434)
	at ts (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:190:15080)
	at MessagePortMain.<anonymous> (file:///usr/lib/code/out/vs/workbench/api/node/extensionHostProcess.js:190:11628)
	at MessagePortMain.emit (node:events:518:28)
	at MessagePortMain._internalPort.emit (node:electron/js2c/utility_init:2:2949) setContext undefined
2025-02-23 23:19:17.748 [info] Extension host with pid 3674 exiting with code 0
